src/sass/components/_steps.scss

```diff
@@ -0,0 +1,13 @@
+// vertical path bar overrides
+.slds-progress_vertical {
+  .slds-progress__item {
+    &:before,
+    &:after {
+      content: none;
+    }
+  }
+}
+
+.slds-progress__item.slds-is-active .slds-progress__marker {
+  border-color: $brand-primary-active;
+}
```

Comment:
@stacyk This looks to my eye like a _very_ slight difference from the default -- is there a reason we want to override it?